tools/test-harnesses/src/tests/queryByLgId.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { queryByLgId } from '../utils/getQueries';
+
+describe('queryByLgId', () => {
+  test('gets element with id', () => {
+    render(<div data-lgid="testing-id">test 1</div>);
+    const element = queryByLgId('testing-id');
+    expect(element).toBeInTheDocument();
+  });
+
+  test('clears the DOM', () => {
+    expect(document.body).toBeEmptyDOMElement();
+  });
```

Comment:
what's this test doing?

Reply:
This just seems like something we don't need to be testing ourselves, since we're not doing anything with rendering/cleanup, are we? 
Curious what was breaking this

Reply:
This was an issue when I forgot to update the jest config and tests were not clearing but I guess I can remove it

Reply:
removed!